app/controllers/concerns/remember_device_concern.rb

```diff
@@ -69,16 +69,18 @@ def expired_for_interval?(user, interval)
     )
   end
 
-  def handle_valid_remember_device_cookie
+  def handle_valid_remember_device_cookie(remember_device_cookie:)
     user_session[:auth_method] = TwoFactorAuthenticatable::AuthMethod::REMEMBER_DEVICE
     mark_user_session_authenticated(:device_remembered)
-    handle_valid_remember_device_analytics
+    handle_valid_remember_device_analytics(cookie_created_at: remember_device_cookie.created_at)
     redirect_to after_otp_verification_confirmation_url unless reauthn?
     reset_otp_session_data
   end
 
-  def handle_valid_remember_device_analytics
-    analytics.remembered_device_used_for_authentication
+  def handle_valid_remember_device_analytics(cookie_created_at:)
+    analytics.remembered_device_used_for_authentication(
+      cookie_created_at: cookie_created_at,
+    )
```

Comment:
would it make sense to also log this in a form of like, how many seconds/minutes etc the cookie is valid so we could easily look at distributions in Cloudwatch? (w/o having to parse as timestamp)

Reply:
Yes, that's a much better idea, thank you

Reply:
Amended the changes to include the age of the cookie